LogHammer/Assets/Scripts/BallMovement.cs

```diff
@@ -2,13 +2,19 @@
 using System.Collections.Generic;
 using UnityEngine;
 
+//Handles the movement of the Player
 public class BallMovement : MonoBehaviour
 {
+    //to store the reference of rigidbody component of the player, to which we will apply some force, for its movement
     private Rigidbody rigidBody;
```

Comment:
2d vector? I am using AddForce(vector3 force) method of RigidBody. That is why I took its reference here.